react-frontend/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
We can keep this as it is for now. However, I think we should make sure it is up to date with the project and that the content is relevant to the project further down the road.